### PR TITLE
chore(litmus-portal): Integrate kubera-ui theme provider

### PR DESCRIPTION
**Pull request by saswatamcode**
## Proposed changes
Add base styles for integrating `kubera-ui` theme-provider into litmus-portal. 

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Special notes for your reviewer:
Colours might look off in certain cases as custom theme in no longer used.

### CODE REVIEW

**Comment by ishangupta-ds** [comment]
@saswatamcode build has failed please run `npx browserslist@latest --update-db`

File: litmus-portal/frontend/package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "jsonwebtoken": "^8.5.1",
     "jspdf": "^2.1.1",
     "jspdf-autotable": "^3.5.13",
+    "kubera-ui": "^0.3.2",
```
Comment (S-ayanide) [inline litmus-portal/frontend/package.json:35]: Can you update it to 0.3.4 for all the latest changes to be seamless

**Review by S-ayanide — CHANGES_REQUESTED** [review]

**Comment by S-ayanide** [comment]
> @saswatamcode build has failed please run `npx browserslist@latest --update-db`

This is just a warning, the build has failed due to the time limit being exceeded. It required more time to complete the optimized build because of the size of the project currently. Generally, it times out in 12 minutes or so, we need to increase the timeout to 20 minutes like we did in Chaos.

**Review by S-ayanide — APPROVED** [review]
LGTM except GeoMap

**Review by amityt — APPROVED** [review]

**Pull request merged**